lang/ilias_pl.lang

```diff
@@ -1945,7 +1945,6 @@ auth#:#login_page_switch_rte#:#Użyj edytora sformatowanego tekstu
 auth#:#lti_consumer_inactive#:#Odbiorca LTI jest wyłączony
 auth#:#saml_tab_head_idp#:#IDP
 awrn#:#awareness_now_online#:#Teraz w trybie online
-awrn#:#awareness_now_online_users#:#[ONLINE_USER_NAMES]
```

Comment:
Please only touch the `de` and `en` language files. This change has to be reverted.